cn_indian_payroll/cn_indian_payroll/overrides/salary_slip.py

```
@@ -1220,7 +1203,8 @@ def accrual_update(self):
                         'Employee Benefit Accrual',
                         filters={
                             'salary_slip': self.name,
-                            'salary_component': i.reimbursements
+                            'salary_component': i.reimbursements,
+                            "docstatus": ["in", [0, 1]]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)